tools/mountsnoop.py

```diff
@@ -14,9 +14,11 @@
 import argparse
 import bcc
 from bcc.containers import filter_by_containers, ContainersMap, print_container_info
+from bcc.utils import ArgString, printb, disable_stdout
```

Comment:
`ArgString` and `printb` are not used, right?

Reply:
Good catch!